src/main/java/bio/terra/cda/app/controller/QueryApiController.java

```diff
@@ -89,4 +94,25 @@ private String createNextUrl(String jobId, int offset, int limit) {
 
     return sendQuery(querySql, dryRun);
   }
+
+  @Override
+  public ResponseEntity<QueryResponseData> uniqueValues(String version, @Valid String body) {
+
+    String bqTable = "gdc-bq-sample";
+    Map<String, String> tableParts = QueryTranslator.parseTableName(body);
+    // SELECT * FROM UNNEST (SELECT DISTINCT p.column FROM qualifiedTABLE as p);
+    String querySql =
+        "SELECT DISTINCT "
+            + tableParts.get("columnName")
+            + " FROM "
+            + bqTable
+            + "."
+            + tableParts.get("tableName");
+    logger.info("uniqueValues: " + querySql);
```

Comment:
This looks like a `debug` log, not `info`.

Reply:
fixed